src/qml/controls/ProgressIndicator.qml

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.12
+import QtQuick.Controls 2.12
+
+Control {
+    property real progress: 0
+    Behavior on progress {
+        SmoothedAnimation {
+            velocity: 1
+        }
+    }
+    contentItem: Rectangle {
+        implicitHeight: 6
+        radius: Math.floor(height / 2)
+        color: "#404040"
```

Comment:
Could we avoid magic numbers?

Reply:
We need to move these constants somewhere, something to follow up.